openvdb_ax/codegen/StandardFunctions.cc

```diff
@@ -353,19 +353,82 @@ struct Rand : public FunctionBase
 {
     DEFINE_IDENTIFIER_DOC("rand", "Creates a random number based on the provided "
         "seed. The number will be in the range of 0 to 1. The same number is "
-        "produced for the same seed.")
+        "produced for the same seed. Note that if rand is called without a seed "
+        "the previous state of the random number generator is advanced for the "
+        "currently processing point. This state is determined by the last call to "
```

Comment:
Could be a voxel rather than a point, might be better to say "current processing element"?  Or whatever our generic term for point/voxel is.

Reply:
done